<launch>
  <node pkg="tsukuba_challenge" name="publisher" type="rosbag_publisher.py" />

  <node pkg="tsukuba_challenge" name="listener" type="rosbag_listener.py" />

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/device_0/sensor_1/Color_0/image/data" />
  </node>
</launch>
